model/pytorch/tests/test_pytorchnet.py

```diff
@@ -149,20 +152,43 @@ async def test_02_predict(self):
         self.assertIn(results["value"], self.model.config.classifications)
         self.assertTrue(results["confidence"])
 
+    async def test_03_tune(self):
+        acc = await tune(
+            self.model,
+            self.tuner,
+            self.scorer,
+            Feature("label", str, 1),
+            [DirectorySource(
+                foldername=str(self.traindir) + "/rps",
+                feature="image",
+                labels=["rock", "paper", "scissors"],
+            )],
+            [DirectorySource(
+                foldername=str(self.testdir) + "/rps-test-set",
+                feature="image",
+                labels=["rock", "paper", "scissors"],
+            )],
+        )
+        self.assertGreater(acc, 0.0)
+
```

Comment:
Okay so heres the drill, normally we won't want to test tuners for each model unless they are behaving differently for each model. 
We need to add the unit tests in their respective tuner modules.
ie. you add a test for each tuner in their respective directories.
Even if they do behave differently for each model, you add a unit test that loops through all models available an tunes them.

Also a reasonable way to check if tuner is performing well would be to see if it is actually tuning. Eg. for parameter grid you can have two or 3 sets of values. Have one of them to be optimal and the other 2 as absurd values that would throw predictions off. Place a check for optimized parameters.
Might still probably end up `self.assertGreater(acc, 0.0)` for the random search and similar tuners but you get the idea, lets see if you can derive anythign else.


Let's discuss these further in a meeting with @programmer290399 and figure out the best way to set these up.